### ISSUE TITLE
WordCamp Sevilla 2021

### ISSUE THREAD

**Issue by abhansnuk** [wpmarketingteam/Marcomms-Rolling-tasks#71]
Request came in 2021/11/21

**Comment by murillol** [comment]
[Ana's Reply](https://wordpress.slack.com/archives/C0GKJ7TFA/p1637589490376000?thread_ts=1637491139.374300&cid=C0GKJ7TFA):  _Thanks channel for your help. So at first I can't think of many things and perhaps you are more used to knowing how to spread WordCamps from other communities.
At first, three actions come to mind:_

_1. Write a post on the global blog as the news of the first post-pandemic face-to-face WC.
2. Second a couple of tweets about this WC encouraging you to attend.
3. And finally retweet our important publications as Relevant News._

_What do you think? Thanks in advance._

I think we could work on:

1. Mini social campaign (in English) like we do for other teams, highlighting the fact that it's in-person and they're sharing their journey while organising it (through livestreams, etc), and also a call to action to visit the site and attend if it's possible. 

2. I like the Post idea (for our own P2) maybe it could also include a small interview about how they are organising it, tips for others who want to start up their local WordCamps in person again, how they are encouraging people to get tickets etc?

3. Since we don't really retweet things in languages other than English, it might be tricky to RT items in Spanish, but I think our social campaign should make up for this. 

4. Organise a brainstorm regarding this topic during an upcoming Marketing meeting?

**Comment by murillol** [comment]
**Update**: Ana has indicated there is a Spanish language video and whether we can share. I've replied, we're waiting on the link

**Comment by murillol** [comment]
**Update**: there is a video in English which Ana has shared. I've added subtitles to it and will include in promos.

**Comment by murillol** [comment]
Suggested Posts:

**6 December 2021 - Twitter** 
On 11 December 2021, WordCamp Sevilla will take place, in-person! Meet new people, catch up with familiar faces, and experience the magic of the #WordPress community! As @rociovaldi says, everyone is welcome! [include uploaded video] https://sevilla.wordcamp.org/2021/entradas/ #WCSevilla


**7 December 2021 - LinkedIn** 
Sevilla's in-person WordCamp is back! On 11 December 2021, WordCamp Sevilla will take place.
It will be full of networking activities where you can meet new people, catch up with familiar faces, and experience the magic of the #WordPress community!  
Tickets are almost sold out, so get yours quickly and be sure to follow the #WCSevilla hashtag this week and during the event. 
https://sevilla.wordcamp.org/2021/entradas/
[include uploaded video] 

**8 December 2021 - Facebook**
Sevilla's in-person WordCamp is back and taking place on 11 December 2021!
It will have multiple sessions as well as a lot of networking activities where you can meet new people, catch up with familiar faces, and experience the magic of the #WordPress community!  

Tickets are almost sold out, so get yours quickly and be sure to follow the #WCSevilla hashtag this week and during the event. 

https://sevilla.wordcamp.org/2021/entradas/
[include uploaded video] 

@abhansnuk please check these ASAP - this is the final week of promos !

**Comment by abhansnuk** [comment]
All done on the 6th. Did the blog go on the WordCamps news?

**Comment by murillol** [comment]
Thanks @abhansnuk! I understand that Ana is working on a post for the P2 (pre-event) that was requested by Rocio, but I am not involved in that part. I have an additional question: would it be possible to put something up (or retweet/share) on the WordCamp Central Twitter / Facebook accounts regarding the event?

**Comment by abhansnuk** [comment]
I can ask. Is there a link to any post they have already done on the WordCamp news site @murillol? Thanks

**Comment by murillol** [comment]
> I can ask. Is there a link to any post they have already done on the WordCamp news site @murillol? Thanks

Thank you! No posts yet on the WC News site but I will keep an eye out and also ask if its coming.  The only link we have besides the event website is: https://central.wordcamp.org/wordcamps/wordcamp-sevilla-5/
